src/nxdoc/nuxeo-server/administration/observability/logs.md

```diff
@@ -0,0 +1,232 @@
+---
+title: Logs
+description: Configuring Nuxeo logs.
+review:
+    date: '2020-06-10'
+    status: requiresUpdates
+    comment: 'To be reviewed since log4j2?'
+labels:
+    - metrics
+    - monitoring
+toc: true
+tree_item_index: 200
+history:
+  -
+---
+Nuxeo logging is compliant with common Java logging frameworks&nbsp;[Log4J](https://logging.apache.org/log4j/1.2/),&nbsp;[Log4j2](https://logging.apache.org/log4j/2.x/),&nbsp;[SLF4J](http://www.slf4j.org/)&nbsp;and&nbsp;[JUL](http://download.oracle.com/javase/6/docs/technotes/guides/logging/index.html).
```

Comment:
There's also [JCL](https://commons.apache.org/proper/commons-logging/) framework in our stack. It was the preferred one before the Log4j2 switch.

Reply:
ok inserted JCL between log4j2 and SLF4J